spec/support/github_service_helper.rb

```diff
@@ -0,0 +1,217 @@
+require 'octokit'
+require 'github_service'
+require 'github_service/issue'
+
+module GithubServiceHelper
+  module_function
```

Comment:
I have never seen this before.  Really interesting.  Not sure I like or not since lately I don't even like bare `private`/`protected` calls either.

Reply:
Oh weird...it creates the following methods as both static methods on the module *and also* private instance methods when mixed in.

Reply:
@Fryguy yeah, this is pure `Ruby` syntax, similar to how `FileUtils` works where you can either use the module directly, or include it in a class and they become private methods for internal use only.

Reply:
@Fryguy yeah, this is pure `Ruby` syntax, similar to how `FileUtils` works where you can either use the module directly, or include it in a class and they become private methods for internal use only.